graphicsmagick/src/wand/magick.rs

```diff
@@ -4584,29 +4670,47 @@ mod tests {
         mw.get_image_page().unwrap();
     }
 
-    fn test_magick_wand_get_image_pixels_inner<T>()
+    fn test_magick_wand_get_set_image_pixels_inner<T>()
     where
-        T: MagickWandExportType + std::fmt::Debug,
+        T: MagickWandExportType + std::fmt::Debug + std::cmp::PartialEq,
     {
         let mut mw = new_logo_magick_wand();
 
-        let pixels = mw.get_image_pixels::<T>(0, 0, 0, 0, "RGBA").unwrap();
+        let columns = mw.get_image_width();
+        let rows = mw.get_image_height();
 
+        // Some basic tests tests
+        let pixels = mw.get_image_pixels::<T>(0, 0, 0, 0, "RGBA").unwrap();
         assert!(pixels.is_empty());
 
         let pixels = mw.get_image_pixels::<T>(0, 0, 10, 10, "RGBA").unwrap();
-
         assert!(!pixels.is_empty());
+
+        // Round trip tests
+        for map in ["RGB", "RGBA"] {
+            let pixels = mw.get_image_pixels::<T>(0, 0, columns, rows, map).unwrap();
+
+            let import = MagickWandImportSlice::new(columns, rows, map, &pixels).unwrap();
+
+            let mut wand = new_logo_magick_wand();
```

Comment:
This is probably not the best way to do it.
I don't know how to create a new `wand` containing an empty image.

Reply:
@jmjoy Do you know how to create an empty image suitable for importing?
It seems that just calling `MagickWand::new` is not enough for this.

Reply:
Use [DrawingWand::rectangle](https://docs.rs/graphicsmagick/latest/graphicsmagick/wand/drawing/struct.DrawingWand.html#method.rectangle)?

Reply:
Hmmm, I will have a try.

Reply:
@jmjoy That didn't work.

Reply:
:cry: